Makefile

```diff
@@ -68,6 +68,8 @@ SAIL_SYS_SRCS += riscv_sys_exceptions.sail  # default basic helpers for exceptio
 SAIL_SYS_SRCS += riscv_sync_exception.sail  # define the exception structure used in the model
 SAIL_SYS_SRCS += riscv_next_control.sail    # helpers for the 'N' extension
 SAIL_SYS_SRCS += riscv_softfloat_interface.sail riscv_fdext_regs.sail riscv_fdext_control.sail
+SAIL_SYS_SRCS += softfloat/riscv_softfloat_common.sail
```

Comment:
I would call this `float/riscv_float_common.sail` rather than `softfloat`. When we've fully migrated away from the C softfloat library it wouldn't really make sense to talk about softfloat here.

Reply:
Sure.